src/i18n/pt-br/index.js

```diff
@@ -0,0 +1,360 @@
+        reset: 'Reiniciar',
+        gas_used: 'Gás usado',
+        transactions: 'Transações',
+        account: 'Conta',
+        contract: 'Contrato',
+        erc20_transfers: 'Transferências ERC20',
+        erc721_transfers: 'Transferências ERC721',
+        erc1155_transfers: 'Transferências ERC1155',
+        tokens: 'Tokens',
+        created_at_trx: 'Criada na TRX',
+        by_address: 'Por endereço',
+        number_used_once: 'Número usado uma vez (nonce)',
+        native_account: 'Conta nativa',
+        balance: 'Saldo',
+        view_source_prompt: 'Este contrato foi verificado. Você pode ver o código-fonte e os metadados na guia \'contrato\'',
+        account_url: '{ domain }/account/{ account }',
+        tlos_balance: '{ balance } TLOS',
+        couldnt_retreive_metadata_for_address: 'Não foi possível recuperar os metadados para { address }: { message }',
+        transaction_details: 'Detalhes da transação',
+        transaction_not_found: 'Não encontrado: { hash }',
+        general: 'Geral',
+        details: 'Detalhes',
+        logs: 'Logs',
+        internal_txns: 'Transações internas',
+        transaction_hash: 'Hash da transação',
+        click_to_change_date_format: 'Clique para alterar o formato da data',
+        block_number: 'Número do bloco',
+        from: 'De',
+        to: 'Para',
+        date: 'Data',
+        success: 'Sucesso',
+        failure: 'Falha',
+        status: 'Status',
+        balance_gwei: '{ amount } GWEI',
+        balance_tlos: '{ amount } TLOS',
+        error_message: 'Mensagem de erro',
+        contract_function: 'Função do contrato',
+        function_parameters: 'Parâmetros da função',
+        deployed_contract: 'Contrato implantado',
+        click_to_show_in_wei: 'Clique para mostrar em wei',
+        gas_price_charged: 'Preço do gás cobrado',
+        gas_fee: 'Taxa de gás',
+        gas_limit: 'Limite de gás',
+        nonce: 'Nonce',
+        input: 'Entrada',
+        output: 'Saída',
+        value: 'Valor',
+    },
+    components: {
+        verify_prompt: 'Este contrato ainda não foi verificado. Você gostaria de carregar o(s) contrato(s) e os metadados para verificar o código-fonte agora?',
+        verify_contract: 'Verificar contrato',
+        search_evm_address_failed: 'A busca pelo endereço EVM vinculado à conta nativa { accountName } falhou. Você pode criar uma em wallet.telos.net',
+        unknown_web3_login_type: 'Tipo de login web3 desconhecido: { provider }',
+        connect_wallet: 'Conectar carteira',
+        view_address: 'Ver endereço',
+        disconnect: 'Desconectar',
+        disable_wallet_extensions: 'Desative as carteiras de extensão ou defina a Brave Wallet como padrão nas configurações da carteira do navegador para usar a Brave Wallet',
+        enable_wallet_extensions: 'Ative a extensão MetaMask e defina o padrão como `preferir extensões` nas configurações da carteira do navegador para usar a carteira MetaMask',
```

Comment:
@karynemayer, these two lines were recently added in the [Brave native wallet PR](https://github.com/telosnetwork/teloscan/pull/314). 

Can you check them and suggest corrections if needed?